pages/Useful Utilities/File-Managers.md

```diff
@@ -0,0 +1,35 @@
+---
+weight: 10
+title: File Managers
+---
+
+## GUI
+
+- [dolphin](https://github.com/KDE/dolphin): File manager by KDE.
+- [nautilus](https://apps.gnome.org/en/Nautilus/): A file browser for Gnome.
+    - [nautilus-admin-gtk4](https://github.com/MacTavishAO/nautilus-admin-gtk4): Open files with elevated priviledges.
+    - [nautilus-image-converter](https://gitlab.gnome.org/coreyberla/nautilus-image-converter): Resize and rotate images in Nautilus.
+    - [nautilus-open-any-terminal](https://github.com/Stunkymonkey/nautilus-open-any-terminal): Open terminals in selected directory.
+- [nemo](https://github.com/linuxmint/nemo): File browser for Cinnamon.
+    - [nemo-fileroller](https://github.com/linuxmint/nemo-extensions/tree/master/nemo-fileroller): File archiver extension for Nemo.
+    - [nemo-terminal](https://github.com/linuxmint/nemo-extensions/tree/master/nemo-terminal): Embedded terminal window for Nemo.
+- [pcmannfm](https://github.com/lxde/pcmanfm): Extremely fast and lightweight file manager.
+- [thunar](https://github.com/neilbrown/thunar): Modern, fast and easy-to-use file manager for Xfce.
+
+## TUI
+
+- [nnn](https://github.com/jarun/nnn): n³ The Unorthodox terminal file manager.
+- [ranger](https://github.com/ranger/ranger): A VIM-inspired filemanager for the console.
+- [superfile](https://github.com/yorukot/superfile): Pretty fancy and modern terminal file manager.
+- [yazi](https://yazi-rs.github.io/): Blazing fast terminal file manager written in Rust, based on async I/O. 
+    - [nerd-fonts](https://github.com/ryanoasis/nerd-fonts): For icon support.
+    - [ffmpegthubmnailer](https://github.com/dirkvdb/ffmpegthumbnailer): For video thumbnails,
+    - [p7zip(7-zip)](https://github.com/p7zip-project/p7zip): For archive extraction and preview.
+    - [jq](https://jqlang.github.io/jq/): For JSON preview.
+    - [poppler](https://gitlab.freedesktop.org/poppler/poppler): For PDF preview.
+    - [fd](https://github.com/sharkdp/fd): For file searching.
+    - [ripgrep](https://github.com/BurntSushi/ripgrep): For file content searching.
+    - [fzf](https://github.com/junegunn/fzf): For quick subtree navigation.
+    - [zoxide](https://github.com/ajeetdsouza/zoxide): For historical directories navigation.
+    - [wl-clipboard](https://github.com/bugaevc/wl-clipboard): For system clipboard support.
```

Comment:
TUI stuff have tons of extensions, no point in having all of them here, I'd drop all of these, considering they are more like dependencies.

Reply:
done